homeassistant/components/plex/server.py

```diff
@@ -87,6 +61,13 @@ def __init__(self, hass, server_config, known_server_id=None, options=None):
         self._accounts = []
         self._owner_username = None
         self._version = None
+        self._update_platforms_debouncer = Debouncer(
+            hass,
+            _LOGGER,
+            cooldown=DEBOUNCE_TIMEOUT,
+            immediate=True,
+            function=self._async_update_platforms,
+        )
 
```

Comment:
You can add this here to skip 1 await:
```suggestion
        self.async_update_platforms = self._update_platforms_debouncer.async_call
```

You can even not store the debounce class at all, just the method :)